packages/p2p/src/components/modal-manager/modals/rating-modal/rating-modal.jsx

```diff
@@ -68,15 +67,10 @@ const RatingModal = ({
 
 RatingModal.propTypes = {
     is_buy_order_for_user: PropTypes.bool,
-    is_rating_modal_open: PropTypes.bool,
     is_user_recommended_previously: PropTypes.number,
-    onClickClearRecommendation: PropTypes.func,
     onClickDone: PropTypes.func,
```

Comment:
`is_buy_for_user` can't be refactored to the modal since in `order-table-row.jsx` it relies on the local state `order_state` where it is initialised from props `rowRenderer`.
`is_user_recommended_previously` is also passed from `order_state` which is obtained from props `rowRenderer`
`onClickDone` and `onClickSkip` function on `order-table-row.jsx` uses local state `should_show_order_details.current = true`